lib/elixir/src/elixir_module.erl

```diff
@@ -226,11 +231,12 @@ build(Line, File, Module, Lexical) ->
 
 eval_form(Line, Module, Data, Block, Vars, E) ->
   {Value, EE} = elixir_compiler:eval_forms(Block, Vars, E),
-  elixir_overridable:store_pending(Module),
+  OverridableDefinitions = elixir_overridable:store_pending(Module),
   EV = elixir_env:linify({Line, reset_env(EE)}),
   EC = eval_callbacks(Line, Data, before_compile, [EV], EV),
-  elixir_overridable:store_pending(Module),
-  {Value, EC}.
+  OverriddenDefinitions = elixir_overridable:store_pending(Module),
+  OverridablePairsNotOverridden = OverridableDefinitions -- OverriddenDefinitions,
```

Comment:
(You are right, these two lists should be added... I must have been looking at the wrong thing before)